project/Dependencies.scala

```diff
@@ -77,6 +77,7 @@ object Dependencies {
   val scanamo = "com.gu" %% "scanamo" % "0.8.3"
   val enumeratumPlayJson = "com.beachape" %% "enumeratum-play-json" % "1.5.6"
   val commercialShared = "com.gu" %% "commercial-shared" % "6.0.0"
+  val jenkins = "com.gu" %% "jenkins" % "0.1-SNAPSHOT"
```

Comment:
@LATaylor-guardian I'm leaving all doors open, this may well be the solution we opt for eventually. What I'm looking at now is what signature would be suit everyone, e.g.

- a simple function
```
Atom => Rendering
```

- or a paramorphism
```
Atom => (Atom, Rendering)
```

- or a mapping where all atoms are rendered
```
CAPIResponse => RenderedCAPIResponse
```

- or a client API that handles everything
```
CAPIRequest => Future[RendererdCAPIResponse]
```

Here is the time for everyone to voice their opinion 😃